src/Common/Session/EncounterSessionUtil.php

```diff
@@ -0,0 +1,53 @@
+<?php
+
+/**
+ * EncounterSessionUtil refactored from encounter.inc handles setting the encounter in the session
+ *
+ * @package   OpenEMR
+ * @link      http://www.open-emr.org
+ * @author    <Unknown> Authorship was not listed in encounter.inc
+ * @author    Stephen Nielson <stephen@nielson.org>
+ * @copyright Copyright (c) 2018 Brady Miller <brady.g.miller@gmail.com>
+ * @copyright Copyright (c) 2021 Stephen Nielson <stephen@nielson.org>
+ * @license   https://github.com/openemr/openemr/blob/master/LICENSE GNU General Public License 3
+ */
+
+namespace OpenEMR\Common\Session;
+
+use OpenEMR\Services\FormService;
+
```

Comment:
@adunsulag 
Did you miss this class?

Reply:
Ouch! Yes I Did!

Reply:
no worries, we all do it more often then not!